src/build/__init__.py

```diff
@@ -340,17 +351,15 @@ def _handle_backend(self, hook: str) -> Iterator[None]:
         try:
             yield
         except pyproject_hooks.BackendUnavailable as exception:
-            raise BuildBackendException(  # noqa: B904 # use raise from
+            raise BuildBackendException(
                 exception,
                 f"Backend '{self._backend}' is not available.",
                 sys.exc_info(),
-            )
+            ) from None
         except subprocess.CalledProcessError as exception:
-            raise BuildBackendException(  # noqa: B904 # use raise from
-                exception, f'Backend subprocess exited when trying to invoke {hook}'
-            )
+            raise BuildBackendException(exception, f'Backend subprocess exited when trying to invoke {hook}') from None
         except Exception as exception:
-            raise BuildBackendException(exception, exc_info=sys.exc_info())  # noqa: B904 # use raise from
+            raise BuildBackendException(exception, exc_info=sys.exc_info()) from None
```

Comment:
I think these were holdovers from Python 2 days. We are propagating the info anyway, so I think `from None` is correct here.